RELEASENOTES.md

```diff
@@ -6,7 +6,7 @@
 
 ### Fixed
 
-- None
+- Fixed an issue where opening a synchronized Realm that is open in another process would result in an error with a message `History type not consistent`. ([#1593](https://github.com/realm/realm-studio/issues/1593))
```

Comment:
Move to CHANGELOG?

Reply:
I think the way Studio works is you add notes to the RELEASENOTES.md which are then moved to CHANGELOG during the release process.

Reply:
Alrighty 🙂 , would have expected it to be the other way around 😛

Reply:
Yeah, Studio is the only different one and the check isn't correct for the way the repo is setup.